Backend/Data_/UnitOfWork.cs

```diff
@@ -10,10 +10,11 @@ namespace Data_
 {
     public class UnitOfWork: IUnitOfWork
     {
-        private readonly GmfctnContext _Context;
+        public readonly GmfctnContext _Context;
```

Comment:
The Context should be private here, I think you changed it to use it like that UnitOfWork.Context.Users for example, but it's not the good solution so fix that

Reply:
Fixed